### PR TITLE
Removed legacy references

### PR DESCRIPTION
**Pull request by bdunogier**
> Required by ezsystems/ezpublish-community#226

## TODO
- [ ] Before merging, remove the **tmp** commits

### CODE REVIEW

**Comment by yannickroger** [comment]
+1 (if travis failure is ok)

**Comment by bdunogier** [comment]
I've looked into the issue that's showing up, and gotta say I could reproduce it locally, on a version that doesn't have legacy.

I'll try with a version with legacy, but it will take a bit more time...

**Comment by bdunogier** [comment]
Changed branch to the required one, tests should pass now.

This will have to be merged at the same time than https://github.com/ezsystems/ezpublish-community/pull/226.

File: Context/EzContext.php [hunk 1]
```diff
@@ -108,13 +108,6 @@ public function prepareKernel( $event )
 
         // Inject a properly generated siteaccess if the kernel is booted, and thus container is available.
         $container->set( 'ezpublish.siteaccess', $this->generateSiteAccess() );
-
-        // Replacing legacy kernel handler web by the CLI one
-        // @todo: this should be somewhat done in the legacy bundle
-        $legacyHandlerCLI = $container->get( 'ezpublish_legacy.kernel_handler.cli' );
```
Comment (andrerom) [inline Context/EzContext.php]: What about making this detect legacy instead, especially if we end up with some tests still running using legacy.

Comment (bdunogier) [inline Context/EzContext.php]: Maybe. My goal for now is to get the tests to pass, but we can probably come up with stgh a bit more conditional, yes.

Comment (andrerom) [inline Context/EzContext.php]: well if we get them to pass w/o, then why not, but if we have to remove tons of tests to do that it is kind of a shame.

**Comment by andrerom** [comment]
+1, but we will most likely need to clean this up, 6.0.0-alpha1 tag here.

**Pull request merged**